flash/image/embedding/model.py

```diff
@@ -100,6 +112,15 @@ def __init__(
             learning_rate=learning_rate,
         )
 
+        transform, collate_fn = self.transforms.get(pretraining_transform)(**pretraining_transform_kwargs)
+        to_tensor_transform = ApplyToKeys(
+            DefaultDataKeys.INPUT,
+            transform,
+        )
+
+        self.adapter.set_state(CollateFn(collate_fn))
+        self.adapter.set_state(ToTensorTransform(to_tensor_transform))
```

Comment:
Not sure about this. I think it means that the `pre_tensor_transform` and `post_tensor_transform` etc. provided either by the user or our default transforms will still be running. So you’ll get resize, flip, etc. then the multicrop transform or whatever. Maybe we should add a state thing to allow the model to override the whole transform recipe? How should the VISSL stuff behave with users custom transforms?

Reply:
yeah, ideally the user should not put in their own custom transforms here, so I will add a line to override everything within the model